core/src/main/java/org/javarosa/core/model/QuestionDef.java

```diff
@@ -240,12 +243,10 @@ public void readExternal(DataInputStream dis, PrototypeFactory pf) throws IOExce
         }
         setDynamicChoices((ItemsetBinding)ExtUtil.read(dis, new ExtWrapNullable(ItemsetBinding.class)));
         mQuestionStrings = (Hashtable<String, QuestionString>)ExtUtil.read(dis, new ExtWrapMap(String.class, QuestionString.class));
+        extensions = (Vector)ExtUtil.read(dis, new ExtWrapListPoly(), pf);
     }
 
-    /*
-     * (non-Javadoc)
-     * @see org.javarosa.core.util.Externalizable#writeExternal(java.io.DataOutputStream)
-     */
+    @Override
```

Comment:
You can't do @override declarations in the javarosa codbase due to 1.3 compliance